src/sbm_gwf_model.jl

```diff
@@ -108,6 +97,18 @@ function initialize_sbm_gwf_model(config::Config)
     βₗ = ncread(nc, param(config, "input.lateral.land.slope"); sel = inds, type = Float)
     clamp!(βₗ, 0.00001, Inf)
     ldd_2d = ncread(nc, param(config, "input.ldd"); allow_missing = true)
+
+    # the ldd provided has the Y direction flipped, fix this first
+    ldd_fix_map = Dict{Int8, Int8}(
```

Comment:
This part seems a model case specific solution for the test model of sbm_gwf_model? I guess the ldd values are not correct in our test model, my suggestion is to solve this in the test model, and not in the code.

Reply:
Ah yes, good catch. Somehow I was under the impression that I was adding this to the test code. This shouldn't be here.